demo/notebook/demo.ipynb

```diff
@@ -10,27 +10,57 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Can you remove these execution counts and outputs?

Reply:
added a new notebook after clearing cells